modules/ding_tabroll/js/ding_tabroll.js

```diff
@@ -31,7 +41,17 @@
 
         // Start tabs rotate when mouse is out.
         tabroll.mouseleave(function () {
-          tabroll.tabs().tabs("rotate", switch_speed);
+          tabroll.tabs().tabs('rotate', switch_speed);
+        });
+
+        // Stops tabs rotation when an element within it is in focus.
```

Comment:
Nope, the rotation stops when the element is in focus, just like when the element is hovered
But, its good you wrote because then i could fix this line to "Starts" where it was "Stops" before :D

https://github.com/ding2/ding2/pull/1422/commits/32c2e8db03efec2ef3aea6ac903df82dfe371d86#diff-7c6117d02b02d1afe6b38c5681996c12R52